ios/fastlane/Fastfile

```diff
@@ -98,6 +103,12 @@ platform :ios do
     )
     if options[:submit]
       deploy_testflight
+      download_dsyms(version: get_version_number)
```

Comment:
For release builds the debug symobls come from Apple because we only upload bitcode, the final compilation happens on their side.